src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupItem.md

```diff
@@ -286,7 +286,7 @@ Workload type of the resource. The acceptable values for this parameter are:
 Type: Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.WorkloadType
 Parameter Sets: GetItemsForContainer, GetItemsForVault
 Aliases:
-Accepted values: AzureVM, AzureFiles, MSSQL, FileFolder, SAPHanaDatabase
+Accepted values: AzureVM, AzureFiles, MSSQL, FileFolder
```

Comment:
this value is supported from client side: 
https://github.com/Azure/azure-powershell/blob/main/src/RecoveryServices/RecoveryServices.Backup/Cmdlets/Item/GetAzureRmRecoveryServicesBackupItem.cs#L101

Please submit an [issue](https://github.com/Azure/azure-powershell/issues/new?assignees=&labels=needs-triage%2Cbug&template=1-AZ-BUG-REPORT.yml) and describe if you have further questions.

Reply:
I believe it was there in the AzureRm module and not in the Az module. If you have the link to the Az module's repo that would be very helpful. I tested again and it doesn't list as a valid value on the client-side. 

Still, if this should be opened as an issue then I believe the bullet above should match the list of accepted values for the Parameter WorkloadType. I opened PR #17041 for the same. Please review it here: https://github.com/Azure/azure-powershell/pull/17041

Reply:
> 

The above link is for the Az module. I think developer forgot to update related docs.

Reply:
@HarvestingClouds : Get-AzRecoveryServicesBackupItem cmdlet now support fetching the backup items for SAPHANADatabase WorkloadType. Are you facing any issues with the same in latest 5.1.0 Az.RecoveryServices module ?